### ISSUE TITLE
`useNativeDriver` was not specified.

### ISSUE THREAD

**Issue by SamimSdz** [magicismight/react-native-root-toast#121]
Hi, this is my code with no options :

```
import Toaster from "react-native-root-toast";

const toast = () => {
    Toaster.show("This is a message");
  };
```

and here is result :
![image](https://user-images.githubusercontent.com/29577616/78872399-79f78300-7a5e-11ea-90e0-059229aa3d23.png)
 
as you see there is a warning on my screen and toast doesn't show anything and it is not working

consolse error :
```
Animated: `useNativeDriver` was not specified. This is a required option and must be explicitly set to `true` or `false`
```
RN 0.62.0
React 16.11.0

**Comment by guizhongyun** [comment]
me too

**Comment by sunnylqm** [comment]
Pr is welcomed

**Comment by MadsonChavante** [comment]
me too

**Comment by jhs851** [comment]
+1

**Comment by XYGR** [comment]
me too

**Comment by mtalhagaoglu** [comment]
i am getting same error...

**Comment by sunnylqm** [comment]
Fixed in 3.2.1
credit @tapioko
